framework/test/jest/integration/specs/modules/chain/rounds.spec.js

```diff
@@ -0,0 +1,383 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+describe('rounds', () => {
```

Comment:
This file is too extensive; maybe split in different tests related to different scenarios?